### DISCUSSION TITLE
When will JoinFace 5 Final be released?

### DISCUSSION THREAD

**Discussion by Master-Code-Programmer** [joinfaces/joinfaces#1615] [Q&A] [answered]
We use Spring Boot 3 in its final version. So does JoinFaces in its latest commits in its repository. But not in the versions released on Maven Central. There is only Milestone 5, which uses all the Spring Boot 3 pre-release versions (Milestones and Release Candidates).
Do you have an estimate when you will release a stable build on Maven Central which uses Spring Boot 3 Final?

**Comment by larsgrefer** [comment, marked as answer]
I've just started the Release process. It should be available vom mavenCentral in a few hours.

**Reply by Master-Code-Programmer** [reply]
> I've just started the Release process. It should be available vom mavenCentral in a few hours.

Thanks, the message got somehow lost in my mailbox. I owe you a thank you. This project is great work. Thanks again.
